src/TypeScriptGenerator.ts

```diff
@@ -1084,7 +1084,7 @@ function getDataTypeName(name: string): string {
 }
 
 // Should match FLOW_TRANSFORMS array
-// https://github.com/facebook/relay/blob/v6.0.0/packages/relay-compiler/language/javascript/RelayFlowGenerator.js#L621-L627
+// https://github.com/facebook/relay/blob/v10.0.0/packages/relay-compiler/language/javascript/RelayFlowGenerator.js#L982
```

Comment:
Only updated the url (or maybe shouldn't have?). The transforms array seems to be in sync with `relay-compiler`.